apps/src/javalab/Javalab.js

```diff
@@ -102,6 +103,13 @@ Javalab.prototype.init = function(config) {
 
   registerReducers({javalab});
 
+  if (config.level.editBlocks) {
```

Comment:
Our config variables are not particularly clear. Would you mind adding a short comment here describing what's happening?